lib/cmd/fh2/app/create.js

```diff
@@ -106,18 +106,14 @@ function doCreateNguiFromTemplate(projectId, title, templateId, deployEnv, cb) {
     "connections": []
   };
 
-  fhc.templates.apps({id:templateId}, function(err, template) {
+  fhc.templates.apps({id:templateId, json:true}, function(err, template) {
     if (err) {
       return cb(err);
     }
-    if (!template) {
-      return cb(i18n._('Template not found: ') + templateId);
-    }
-    payload.template = template;
+    payload.template = template[0];
```

Comment:
better check `template` is an array. Also might be better rename it to `templates` instead